### ISSUE TITLE
Mock server/Try It updates

### ISSUE THREAD

**Issue by pamgoodrich** [stoplightio/platform-docs#278]
1. From Caitlin:

yes, this topic just needs a step added after 2 that says something like “Navigate to the published view”

 https://meta.stoplight.io/docs/platform/ZG9jOjM2OTM3Mjg4-work-with-mock-servers#studio-web

Ok and the “Try It” area part might need to be removed - I think its only called Try It within the editor and the mock server doesn’t work in the editor

Slack discussion:

https://stoplight-internal.slack.com/archives/C0350RJLX1N/p1646254028569699

2. Add something to Try It topic about this:
https://www.notion.so/stoplight/Prism-6e7596f0d6134b8b97074cc55c1a47c8

Basically, we to say soemthing about  Hosted Prism returns a `422 Unprocessable Entity` anytime a GET is paired with a request body, because the request body isn’t transmitted by `fetch` and therefore the request fails validation against the spec.

TL;DR - There is inconsistent behavior between Prism CLI and Hosted Prism along with confusing UX resulting in multiple customer complaints related to GETs with Request Bodies
